### PR TITLE
docs: add docs for zip import and export endpoints

### PR DESCRIPTION
**Pull request by pyphilia**
ref #1528

### CODE REVIEW

File: src/services/item/plugins/importExport/schema.ts [hunk 1]
```diff
@@ -1,24 +1,41 @@
...
-      {
-        parentId: customType.UUID(),
-      },
-      {
-        additionalProperties: false,
-      },
+  operationId: 'importZip',
+  tags: ['item', 'import'],
+  summary: 'Import ZIP content',
+  description: 'Import content of a ZIP, creating corresponding structure and items.',
```
Comment (ReidyT) [inline src/services/item/plugins/importExport/schema.ts:13]:
```suggestion
  description: 'Import and extract the content of a ZIP, creating the corresponding structure and items.',
```

**Review by ReidyT — APPROVED** [review]
LGTM, just have a look at my suggestion 🫠

File: src/services/item/plugins/importExport/schema.ts [hunk 2]
```diff
@@ -1,24 +1,41 @@
...
+
+  querystring: customType.StrictObject({
+    parentId: Type.Optional(
```
Comment (spaenleh) [inline src/services/item/plugins/importExport/schema.ts:16]: Interesting that the name of this parameter is not consistent with the file upload one.

File: src/services/item/plugins/importExport/schema.ts [hunk 3]
```diff
@@ -1,24 +1,41 @@
...
-      },
-      {
-        additionalProperties: false,
-      },
+  operationId: 'importZip',
+  tags: ['item', 'import'],
+  summary: 'Import ZIP content',
+  description: 'Import content of a ZIP, creating corresponding structure and items.',
+
+  querystring: customType.StrictObject({
```
Comment (spaenleh) [inline src/services/item/plugins/importExport/schema.ts:15]: There is no handling of "after this element" here ?

File: src/services/item/plugins/importExport/schema.ts [hunk 4]
```diff
@@ -1,24 +1,41 @@
...
+  summary: 'Export content',
+  description:
+    'Export content. Return raw file for single item, or a ZIP with structure and items for a folder.',
+
   params: customType.StrictObject({
     itemId: customType.UUID(),
   }),
+  response: {
+    // return a stream
+    [StatusCodes.OK]: { content: Type.String() },
```
[the remark this answers is not in the document]
Comment (spaenleh) [reply src/services/item/plugins/importExport/schema.ts:38]: Maybe this: https://swagger.io/docs/specification/v3_0/describing-responses/#response-that-returns-a-file

**Review by spaenleh — APPROVED** [review]
Looks good, I have left a few comments here and there

File: src/services/item/plugins/importExport/schema.ts (hunk 4 above)
[the remark this answers is not in the document]
Comment (pyphilia) [reply src/services/item/plugins/importExport/schema.ts:38]: I tried already, also it looks like a pb is that it seems we need to define a format? And also `byte` or `binary` are not handled.. We can have an issue for that.

File: src/services/item/plugins/importExport/schema.ts (hunk 3 above)
Comment (pyphilia) [reply src/services/item/plugins/importExport/schema.ts:15]: No, I think it was a bit of work to handle it. Not everything handles the `previousItemId`.

**Pull request merged**